project-base/storefront/components/providers/AuthorizationProvider.tsx

```diff
@@ -28,8 +28,9 @@ export const useAuthorization = () => {
     const isCompanyUser = isB2B && currentCustomerUser?.companyCustomer;
 
     const canSeePrices = customerUserRoles.includes(TypeCustomerUserRoleEnum.RoleApiCustomerSeesPrices);
-    const canManageUsers = isCompanyUser && customerUserRoles.includes(TypeCustomerUserRoleEnum.RoleApiAll);
-    const canManageCompanyData = !isCompanyUser || customerUserRoles.includes(TypeCustomerUserRoleEnum.RoleApiAll);
+    const canManageUsers = isCompanyUser && customerUserRoles.includes(TypeCustomerUserRoleEnum.RoleApiManageCustomers);
+    const canManageCompanyData =
+        !isCompanyUser || customerUserRoles.includes(TypeCustomerUserRoleEnum.RoleApiManageCustomers);
```

Comment:
ok, I did not want to be that guy, however, here is the place that points to another missing role (`ROLE_API_MANAGE_COMPANY_DATA`?) :smile: Or we could rename the new role to something like `ROLE_API_MANAGE_COMPANY_DATA_AND_CUSTOMERS`. I am not sure whether the implication "can manage company users" -> "can manage company billing data" is ok here. I really do not know, we can talk about this :shrug:

Reply:
created new role ROLE_API_MANAGE_COMPANY_DATA